### PR TITLE
Restore JS tests

### PR DESCRIPTION
**Pull request by fcollonval**
Restore existing JS tests

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -167,8 +167,8 @@ jobs:
         set -eux
         python -m pytest --pyargs jupyterlab_pullrequests --cov jupyterlab_pullrequests --cov-report term-missing:skip-covered --no-cov-on-fail
 
-    - name: Unit Test JS (known to be broken)
-      run: jlpm test || echo "TODO"
+    - name: Unit Test JS
+      run: jlpm test
```
Comment (bollwyvl) [inline .github/workflows/build.yml:171]: :+1:

File: package.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
     "clean:labextension": "rimraf jupyterlab_pullrequests/labextension",
     "eslint": "eslint . --ext .ts,.tsx --fix",
     "eslint:check": "eslint . --ext .ts,.tsx",
-    "test": "jest",
+    "test": "jest --coverage",
```
Comment (bollwyvl) [inline package.json:42]: :drooling_face:

**Review by bollwyvl — APPROVED** [review]
LGTM, merge at your leisure, if you've nothing else planned.

**Comment by bollwyvl** [comment]
Will update my sub-pr to get this proper for lab 3.

**Comment by fcollonval** [comment]
Thanks

**Pull request merged**